code/src/ProjectTemplates/WinUI/CS/CSharp.WinUI.Desktop.Solution/CSharp.WinUI.Desktop.Solution.cs-CZ.vstemplate

```diff
@@ -4,7 +4,7 @@
             xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
   <TemplateData>
     <Name>App (WinUI 3 in Desktop; local)</Name>
-    <Description>Windows Template Studio quickly builds a WinUI Desktop app, using a wizard-based UI to turn your needs into a foundation of Windows 10 patterns and best practices.</Description>
+    <Description>Windows 템플릿 스튜디오는 마법사 기반 UI를 사용하여 Windows 10 패턴 및 모범 사례의 기초로 귀하의 요구를 설정하는 WinUI 데스크톱 앱을 신속하게 빌드합니다.</Description>
```

Comment:
Not cs-CZ (Czech) localization. Same text of ko-KR (Korean)

Reply:
Good catch